main.py

```diff
@@ -239,8 +240,9 @@ def generate_html_report(
     except FileExistsError:
         pass
 
+    fixed_html = transform(html, pretty_print=True)
```

Comment:
premailer moves the styles so they're in the `style` attribute of the relevant tags rather than declared in a `style` tag. This works better with gmail.